Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM elixir:1.9-alpine as builder
+
+LABEL author "Emil Bostijancic <emil.bostijancic@gmail.com>"
+
+ENV MIX_ENV prod
+#ENV REPLACE_OS_VARS true
+
+RUN apk add --no-cache build-base git bash curl ca-certificates
```

Comment:
I think we may not need git, curl, and ca-ceritifcates to build this project, could you double-check?

Reply:
git can be removed, I am not sure about ca-certificates, this is a build container image anyway, so it will not have an impact on the final docker image.